src/main/webapp/app/entities/ace-editor/text-change.model.ts

```diff
@@ -0,0 +1 @@
+export type TextChange = { action: string; start: { row: number; column: number }; end: { row: number; column: number } };
```

Comment:
would it make sense to use a proper class type here?

Reply:
I think in this case TextChange is really just a plain object without any functionality attached to it. So I would suggest to keep it lightweight in this case.